### DISCUSSION TITLE
Placement in Bid Requests

### DISCUSSION THREAD

**Discussion by kroening-stroeer** [openooh/venue-taxonomy#80] [Q&A] [answered]
Could someone please post an example bid request taking use of the 1.1 version taxonomy? In other words how to use the new extension.

many thanks
Sebastian

**Comment by jayshao** [comment, marked as answer]
Sorry for the poor responsiveness. With OpenRTB releasing a 2.6-202211 release that should include native support for DOOH, I think we'll have better samples shortly. The current draft is in a public branch at: https://github.com/InteractiveAdvertisingBureau/openrtb2.x/blob/OOH-Ammends/implementation.md#796---dooh-example-scenarios
